src/main/java/com/yegor256/Jhome.java

```diff
@@ -70,6 +70,26 @@ public Path path(final String loc) {
         return this.home.resolve(loc);
     }
 
+    /**
+     * Find the {@code java} binary.
+     * @return The path of it
+     */
+    public Path java() {
+        return this.home.resolve("bin").resolve("java");
```

Comment:
@volodya-lombrozo looks like code duplication to me. It's better to call `path("bin/java")` from here